Cannot compile with NGINX 1.11.0

With the latest NGINX 1.11.0, building fails.

> cc -c -pipe  -O -W -Wall -Wpointer-arith -Wno-unused-parameter -Werror -g -DTCP_FASTOPEN=23 -O3 -march=native -flto -DNDK_SET_VAR -DNDK_UPSTREAM_LIST -I/usr/local/src/nginx-rtmp-module  -I src/core -I src/event -I src/event/modules -I src/os/unix -I /usr/local/src/ngx_brotli_module/brotli_encoder -I /usr/local/src/ngx_devel_kit/objs -I objs/addon/ndk -I /usr/local/src/openssl-1.0.2h/.openssl/include -I objs -I src/http -I src/http/modules -I src/http/v2 -I /usr/local/src/ngx_devel_kit/src -I /usr/local/src/ngx_devel_kit/src -I /usr/local/src/ngx_devel_kit/objs -I objs/addon/ndk -I /usr/local/src/nginx-rtmp-module \
	-o objs/addon/nginx-rtmp-module/ngx_rtmp_core_module.o \
	/usr/local/src/nginx-rtmp-module/ngx_rtmp_core_module.c
/usr/local/src/nginx-rtmp-module/ngx_rtmp_core_module.c: In function 'ngx_rtmp_core_listen':
/usr/local/src/nginx-rtmp-module/ngx_rtmp_core_module.c:561:9: error: cannot convert to a pointer type
         if (ngx_memcmp(ls[i].sockaddr + off, u.sockaddr + off, len) != 0) {
         ^
In file included from src/core/ngx_core.h:51:0,
                 from /usr/local/src/nginx-rtmp-module/ngx_rtmp_core_module.c:8:
src/core/ngx_string.h:102:42: error: incompatible type for argument 2 of 'memcpy'
 #define ngx_memcpy(dst, src, n)   (void) memcpy(dst, src, n)
                                          ^
/usr/local/src/nginx-rtmp-module/ngx_rtmp_core_module.c:581:5: note: in expansion of macro 'ngx_memcpy'
     ngx_memcpy(ls->sockaddr, u.sockaddr, u.socklen);
     ^
In file included from src/os/unix/ngx_linux_config.h:27:0,
                 from src/core/ngx_config.h:26,
                 from /usr/local/src/nginx-rtmp-module/ngx_rtmp_core_module.c:7:
/usr/include/string.h:46:14: note: expected 'const void * __restrict__' but argument is of type 'ngx_sockaddr_t'
 extern void *memcpy (void *__restrict __dest, const void *__restrict __src,
              ^
objs/Makefile:1890: recept voor doel 'objs/addon/nginx-rtmp-module/ngx_rtmp_core_module.o' is mislukt
make[1]: *** [objs/addon/nginx-rtmp-module/ngx_rtmp_core_module.o] Fout 1

uname -a
Linux vps 4.5.0-0.bpo.2-amd64 #1 SMP Debian 4.5.3-2~bpo8+1 (2016-05-13) x86_64 GNU/Linux

gcc -v
gcc version 4.9.2 (Debian 4.9.2-10) 

./configure --prefix=/opt/nginx --user=www-data --group=www-data --add-module=/usr/local/src/ngx_brotli_module --with-http_v2_module --with-http_ssl_module --with-openssl=/usr/local/src/openssl-1.0.2h --with-openssl-opt="enable-ec_nistp_64_gcc_128 threads" --with-md5=/usr/local/src/openssl-1.0.2h --with-md5-asm --with-sha1=/usr/local/src/openssl-1.0.2h --with-sha1-asm --with-pcre-jit --with-file-aio --with-http_flv_module --with-http_geoip_module --with-http_mp4_module --with-http_realip_module --with-http_stub_status_module --with-threads --with-ipv6 --add-module=/usr/local/src/headers-more-nginx-module --add-module=/usr/local/src/echo-nginx-module --add-module=/usr/local/src/ngx_http_substitutions_filter_module --add-module=/usr/local/src/srcache-nginx-module --add-module=/usr/local/src/redis2-nginx-module --add-module=/usr/local/src/ngx_http_redis-0.3.8 --add-module=/usr/local/src/ngx_devel_kit --add-module=/usr/local/src/set-misc-nginx-module --add-module=/usr/local/src/nginx-rtmp-module --with-cc-opt="-DTCP_FASTOPEN=23 -O3 -march=native -flto" --with-ld-opt="-DTCP_FASTOPEN=23 -O3 -march=native -flto"

Am I doing something wrong? Thanks!

--- a/ngx_rtmp_core_module.c
+++ b/ngx_rtmp_core_module.c
@@ -557,7 +557,11 @@ ngx_rtmp_core_listen(ngx_conf_t *cf, ngx_command_t *cmd, void *conf)
             break;
         }
 
+#if (nginx_version >= 1011000)
+        if (ngx_memcmp(ls[i].sockaddr + off, &u.sockaddr + off, len) != 0) {
+#else
         if (ngx_memcmp(ls[i].sockaddr + off, u.sockaddr + off, len) != 0) {
+#endif
             continue;
         }
 
@@ -577,7 +581,11 @@ ngx_rtmp_core_listen(ngx_conf_t *cf, ngx_command_t *cmd, void *conf)
 
     ngx_memzero(ls, sizeof(ngx_rtmp_listen_t));
 
+#if (nginx_version >= 1011000)
+    ngx_memcpy(ls->sockaddr, &u.sockaddr, u.socklen);
+#else
     ngx_memcpy(ls->sockaddr, u.sockaddr, u.socklen);
+#endif
 
     ls->socklen = u.socklen;
     ls->wildcard = u.wildcard;

